usr/share/rear/prep/Linux-i386/330_set_efi_arch.sh

```diff
@@ -0,0 +1,19 @@
+
+# Set EFI architecture, used as suffix for various files in the ESP
+# See https://github.com/rhboot/shim/blob/main/Make.defaults
+
+# Se the variables even if USING_UEFI_BOOTLOADER empty or no explicit 'true' value
+
+case "$REAL_MACHINE" in
+    (i686|i586|i386)
+        # all these behave exactly like i386.
+        EFI_ARCH=ia32
```

Comment:
It looks like a typo that EFI_ARCH is set to ia32
when REAL_MACHINE is i686 or i586 or i386.
If it is not a typo, please add a comment that explains it.

Reply:
@jsmeix why should it be a typo? Have a look at https://github.com/rhboot/shim/blob/5914984a1ffeab841f482c791426d7ca9935a5e6/Make.defaults#L34 , already linked above.

Reply:
I am sorry - I didn't look carefully.
The leading 'ia' let me falsely think about Itanium
which is ia64 while ia32 is the commonly called i386.